ui/components/app/cancel-speedup-popover/cancel-speedup-popover.js

```diff
@@ -97,11 +92,11 @@ const CancelSpeedupPopover = () => {
     >
       <AppLoadingSpinner className="cancel-speedup-popover__spinner" />
       <div className="cancel-speedup-popover__wrapper">
-        <Typography
-          boxProps={{ alignItems: AlignItems.center, display: DISPLAY.FLEX }}
-          variant={TypographyVariant.H6}
-          marginTop={0}
-          marginBottom={2}
+        <Text
+          variant={TextVariant.bodySm}
+          className="cancel-speedup-popover__description"
+          // gets passed through to Box component
+          display={DISPLAY.FLEX}
         >
           {t('cancelSpeedUpLabel', [
             <strong key="cancelSpeedupReplace">{t('replace')}</strong>,
```

Comment:
Just below this section I think can be improved using some design system components are removing more html bloat. Instead of 
```
              <Box>
                {t('cancelSpeedUpTransactionTooltip', [
                  editGasMode === EditGasModes.cancel
                    ? t('cancel')
                    : t('speedUp'),
                ])}
                <div>
                  <a
                    href="https://community.metamask.io/t/how-to-speed-up-or-cancel-transactions-on-metamask/3296"
                    target="_blank"
                    rel="noopener noreferrer"
                  >
                    {t('learnMoreUpperCase')}
                  </a>
                </div>
              </Box>
```
We could do 
```
              <>
                <Text>
                  {t('cancelSpeedUpTransactionTooltip', [
                    editGasMode === EditGasModes.cancel
                      ? t('cancel')
                      : t('speedUp'),
                  ])}
                </Text>
                <ButtonLink
                  href="https://community.metamask.io/t/how-to-speed-up-or-cancel-transactions-on-metamask/3296"
                  target="_blank"
                >
                  {t('learnMoreUpperCase')}
                </ButtonLink>
              </>
```
Will need to import `ButtonLink` from `component-library/` folder